fmriprep/workflows/bold/confounds.py

```diff
@@ -307,8 +317,8 @@ def init_ica_aroma_wf(name='ica_aroma_wf', ignore_aroma_err=False):
             CSV of noise components identified by ICA-AROMA
         melodic_mix
             FSL MELODIC mixing matrix
-        nonaggr_denoised_file
-            BOLD series with non-aggressive ICA-AROMA denoising applied
+        bold_mni_smooth_nonaggr_denoised
```

Comment:
renamed the denoised files to longer (but more descriptive/discernible names) 
`bold_mni_smooth_nonaggr_denoised`
`bold_mni_nonaggr_denoised`
`bold_t1_nonaggr_denoised`
`bold_nonaggr_denoised`